core-ui/src/components/Extensibility/components/Widget.js

```diff
@@ -14,14 +15,19 @@ export function InlineWidget({ children, structure, ...props }) {
 export function Widget({ structure, value, inlineRenderer, ...props }) {
   const InlineRenderer = inlineRenderer || SimpleRenderer;
   const { Plain, Text } = widgets;
+  const { i18n } = useTranslation();
 
   if (Array.isArray(structure)) {
     return (
       <Plain value={value} structure={{ children: structure }} {...props} />
     );
   }
 
-  const childValue = structure.path ? getValue(value, structure.path) : value;
+  let childValue = structure.path ? getValue(value, structure.path) : value;
```

Comment:
I think it would be better to move it to if/else below. There is no need to calculate the value here if we want to overwrite it one line later.

Reply:
We use this childValue to calculate formula